src/AbstractEnumMapping.php

```diff
@@ -138,6 +138,15 @@ public function remapKeys(string $enumClass, array $arr): array
         return $remapped;
     }
 
+    public function optionalEnumToString(?UnitEnum $enum): ?string
```

Comment:
Je to trochu hnidopisske, ale optional znamena, ze to nekam nemusis davat, takze to evokuje, ze lze metodu zavolat bez argumentu, coz neni pravda. Prefix metody se vztahuje k argumentu, ktery neni optional, ale nullable, takze by se asi mela metoda spise jmenovat `nullableEnumToString`.